danielMunteanu/src/main/java/danielCollectionTask/Address.java

```diff
@@ -0,0 +1,34 @@
+package danielCollectionTask;
+
+public class Address {
+    final String country;
+    private String city;
+    final String street;
```

Comment:
why street can not be modified meanwhile the city can?

Reply:
fixed, both private